docs/reference/migration/migrate_7_0/search.asciidoc

```diff
@@ -306,6 +306,14 @@ Lucene 8 introduced more constraints on similarities, in particular:
 - scores must not decrease when term freq increases,
 - scores must not increase when norm (interpreted as an unsigned long) increases.
 
+[discrete]
+==== Change of scoring for fields that omit norms
+Before, if a field was indexed without norms, `BM25` similarity assumed a document
+length to be `0`, and acted as if its `b` parameter is equal to `0`.
```

Comment:
I find saying the document length is 0 to be a little confusing, because that would imply `avgdl` is 0 and we would be dividing by 0. Maybe we say something like "drops the document length from the calculation, equivalent to setting `b=0`".

Also, maybe we could clarify that this change could affect all similarities. It is really helpful we highlight the specific `BM25` behavior though, as the widely-used default.